### PR TITLE
Create Scopes component using material-ui checkboxes

### PR DESCRIPTION
**Pull request by melyxlin**
Used same formatting as the previous components

### CODE REVIEW

**Review by leorcodes — APPROVED** [review]
LGTM

File: web/client/src/App.css [hunk 1]
```diff
@@ -31,7 +33,7 @@ label {
   flex-direction: row;
   justify-content: flex-end;
   text-align: right;
-  width: 50px;
+  width: 70px;
```
Comment (yousefa00) [inline web/client/src/App.css]: We should try and use rem or % values here to support different screen sizes

File: web/client/src/components/Scopes.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { MDBContainer, MDBCol, MDBBtn, MDBInput } from "mdbreact";
+
+/**
+ *
+ * Class that represents the page that contains the ability to choose the scopes
+ */
+class Scopes extends React.Component {
```
Comment (yousefa00) [inline web/client/src/components/Scopes.js]: I think we're using functional react for this project

File: web/client/src/__tests__/Scopes.test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Enzyme, { mount } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Scopes from "../components/Scopes";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+// test applications use-cases form user's pov. Users access information on a web page and interact with available controls
```
Comment (yousefa00) [inline web/client/src/__tests__/Scopes.test.js]: Typo `form` => `from`

File: web/client/src/__tests__/Scopes.test.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+// assert pm react dom state
+// shawllow rendering is not used for this as we need to be able to test the child components
```
Comment (yousefa00) [inline web/client/src/__tests__/Scopes.test.js]: typo `shawllow` => `shallow`

**Review by yousefa00 — CHANGES_REQUESTED** [review]
Looks great, just a few comments

File: web/client/src/components/Scopes.js (hunk 2 above)
Comment (leorcodes) [reply web/client/src/components/Scopes.js]: +1, in order to add state in functional components look into react hooks

Comment (melyxlin) [reply web/client/src/components/Scopes.js]: Done

File: web/client/src/__tests__/Scopes.test.js (hunk 3 above)
Comment (melyxlin) [reply web/client/src/__tests__/Scopes.test.js]: Done

File: web/client/src/__tests__/Scopes.test.js (hunk 4 above)
Comment (melyxlin) [reply web/client/src/__tests__/Scopes.test.js]: Done

File: web/client/src/App.css (hunk 1 above)
Comment (melyxlin) [reply web/client/src/App.css]: done

File: web/client/src/components/Scopes.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { MDBContainer, MDBCol, MDBBtn, MDBInput } from "mdbreact";
+
+/**
+ * @return {div} returns page the has the ability to choose scopes to use
```
Comment (yousefa00) [inline web/client/src/components/Scopes.js:5]: one typo `the` => `that`

**Review by yousefa00 — APPROVED** [review]
Approved, there was just one small typo in `Scopes` component

**Pull request merged**